test/tests/ver-1dd/comparison_ver-1dd.py

```diff
@@ -0,0 +1,67 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+import os
+import git
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+os.chdir(os.path.dirname(__file__))
+
+num_summation_terms = 1000
+N_o = 3.1622e22             # For convenience lattice density chosen as 3.1622e22 atom/m^3
+c_o = 0.0001                # dissolved gas atom fraction (-)
+D = 1                       # diffusivity (m^2/s)
+l = 1                       # slab thickness (m)
+
+# Extract data from 'gold' TMAP8 run
+tmap_sol = pd.read_csv(os.path.join(git.Repo('.',search_parent_directories=True).working_tree_dir, "test/tests/ver-1dd/gold/ver-1dd_out.csv"))
+tmap_time = np.array(tmap_sol['time'])
+tmap_prediction = np.array(tmap_sol['scaled_outflux'])
+idx = np.where(tmap_time >= 0.01)[0][0]
+
+# Calculate the breakthrough time from numberical solution
```

Comment:
```suggestion
# Calculate the breakthrough time from numerical solution
```

Reply:
If this is an error due to copy pasting, please correct the original .py file as well